src/txorphanage.cpp

```diff
@@ -159,7 +160,7 @@ void TxOrphanage::AddChildrenToWorkSet(const CTransaction& tx)
             for (const auto& elem : it_by_prev->second) {
                 // Get this source peer's work set, emplacing an empty set if it didn't exist
                 // (note: if this peer wasn't still connected, we would have removed the orphan tx already)
-                std::set<uint256>& orphan_work_set = m_peer_work_set.try_emplace(elem->second.fromPeer).first->second;
+                std::set<Txid>& orphan_work_set = m_peer_work_set.try_emplace(elem->second.fromPeer).first->second;
```

Comment:
nit: That's a lot of `first`'s and `second`'s. This PR doesn't make it worse, but if you wanna sneak in some readability improvements I'd very happily support that, e.g.

<details>
<summary>git diff on 68b9c2f268e7a924f4c9182ab1b99c30af86f5d7</summary>

```diff
diff --git a/src/txorphanage.cpp b/src/txorphanage.cpp
index d4c07b0543..80ae4af662 100644
--- a/src/txorphanage.cpp
+++ b/src/txorphanage.cpp
@@ -158,13 +158,16 @@ void TxOrphanage::AddChildrenToWorkSet(const CTransaction& tx)
         const auto it_by_prev = m_outpoint_to_orphan_it.find(COutPoint(tx.GetHash(), i));
         if (it_by_prev != m_outpoint_to_orphan_it.end()) {
             for (const auto& elem : it_by_prev->second) {
+                const auto& txid = elem->first;
+                const auto& orphan_tx = elem->second;
                 // Get this source peer's work set, emplacing an empty set if it didn't exist
                 // (note: if this peer wasn't still connected, we would have removed the orphan tx already)
-                std::set<Txid>& orphan_work_set = m_peer_work_set.try_emplace(elem->second.fromPeer).first->second;
+                auto [it, inserted] = m_peer_work_set.try_emplace(orphan_tx.fromPeer);
+                std::set<Txid>& orphan_work_set = it->second;
                 // Add this tx to the work set
-                orphan_work_set.insert(elem->first);
+                orphan_work_set.insert(txid);
                 LogPrint(BCLog::TXPACKAGES, "added %s (wtxid=%s) to peer %d workset\n",
-                         tx.GetHash().ToString(), tx.GetWitnessHash().ToString(), elem->second.fromPeer);
+                         tx.GetHash().ToString(), tx.GetWitnessHash().ToString(), orphan_tx.fromPeer);
             }
         }
     }

```
</details>

Reply:
wdym readability. You look up the `elem` corresponding to the outpoint in this map, and `elem` refers to an iterator into another map, which you dereference to get the value of the `OrphanTx` object, so that you can extract the key to look up in a third map, so that you can obtain a reference to the element you just inserted or was already there.

Reply:
Not gonna touch that in this PR